packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission.jsx

```diff
@@ -142,7 +142,6 @@ const POISubmission = ({
 
                     return (
                         <OnfidoUpload
```

Comment:
shall we rename the component `OnfidoSdkViewContainer` at `packages/account/src/Sections/Verification/ProofOfIdentity/onfido-sdk-view-container.tsx` to `OnfidoUpload`?
we are exporting and importing with different names..